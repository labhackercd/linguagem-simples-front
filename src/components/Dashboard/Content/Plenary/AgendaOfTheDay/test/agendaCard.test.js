import React from 'react';
import { render, screen } from '@testing-library/react';
import {shallow} from "enzyme/build";
import { mount} from 'enzyme';
import AgendaCard from './../agendaCard'


it("should render the AgendaCard component and match snapshot ", () => {
   
      const pautaInfo = {
        "regime": "Matéria Sobre a Mesa",
        "codRegime": 99,
        "ordem": 1,
        "proposicao_": {
          "id": 2260444,
          "uri": "https://dadosabertos.camara.leg.br/api/v2/proposicoes/2260444",
          "siglaTipo": "REQ",
          "codTipo": 392,
          "numero": 2144,
          "ano": 2020,
          "ementa": "Requer a urgência na tramitação e a imediata inclusão na Ordem do Dia do Projeto de Lei nº 6.537/2019, no Plenário da Câmara dos Deputados."
        },
        "uriProposicaoRelacionada": "https://dadosabertos.camara.leg.br/api/v2/proposicoes/2234707/",
        "situacaoItem": "Retirado",
        "uriVotacao": ""
      };
    
    const sessionIdDadosAbertos = 59896;

    const component = shallow(<AgendaCard pautaInfo={pautaInfo} sessionIdDadosAbertos={sessionIdDadosAbertos}/>);

    expect(component.exists()).toEqual(true);
    expect(component).toMatchSnapshot();
});