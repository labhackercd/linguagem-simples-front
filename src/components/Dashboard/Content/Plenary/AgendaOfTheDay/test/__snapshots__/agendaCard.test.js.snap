// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the AgendaCard component and match snapshot  1`] = `
<Styled(MuiBox)
  width="97%"
>
  <WithStyles(ForwardRef(Paper))
    elevation={0}
  >
    <Styled(MuiBox)
      m={1}
    >
      <WithStyles(ForwardRef(Grid))
        container={true}
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={10}
        >
          <Styled(MuiBox)
            fontWeight="fontWeightRegular"
          >
            <WithStyles(ForwardRef(Typography))
              style={
                Object {
                  "color": "#007E5A",
                }
              }
              variant="h6"
            >
              PL 6407/2013
            </WithStyles(ForwardRef(Typography))>
          </Styled(MuiBox)>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={2}
        >
          <Styled(MuiBox)
            display="flex"
            justifyContent="flex-end"
          >
            <a
              href="https://www.camara.leg.br/proposicoesWeb/fichadetramitacao?idProposicao=593065"
              rel="external noopener noreferrer"
              style={
                Object {
                  "textDecoration": "none",
                }
              }
              target="_blank"
            >
              <WithStyles(ForwardRef(IconButton))
                aria-label="Ir para Proposicao"
                size="small"
              >
                <LaunchIcon
                  fontSize="inherit"
                />
              </WithStyles(ForwardRef(IconButton))>
            </a>
          </Styled(MuiBox)>
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={12}
        >
          <Styled(MuiBox)
            fontSize={11}
          >
            <WithStyles(ForwardRef(Typography))
              style={
                Object {
                  "color": "gray",
                }
              }
            >
              Relator: 
              Laercio Oliveira
               (
              PP        
              -
              SE
              )
            </WithStyles(ForwardRef(Typography))>
          </Styled(MuiBox)>
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </Styled(MuiBox)>
    <WithStyles(ForwardRef(Divider))
      variant="middle"
    />
  </WithStyles(ForwardRef(Paper))>
</Styled(MuiBox)>
`;
